app/models/dimensions/item.rb

```diff
@@ -30,6 +30,10 @@ def promote!(old_item)
     update(latest: true)
   end
 
+  def expanded_links
```

Comment:
Could you please add a unit test for this method? Not sure what happens when `raw_json` is nil...